Sources/FoundationEssentials/JSON/JSONEncoder.swift

```diff
@@ -22,6 +22,11 @@
 #if FOUNDATION_FRAMEWORK
 @_objcRuntimeName(_TtC10Foundation13__JSONEncoder)
 #endif
+
+#if canImport(FoundationInternals)
+package import FoundationInternals
+#endif
+
```

Comment:
Ahhh... I believe you'll have to move this above the `#if FOUNDATION_FRAMEWORK` bit.

Reply:
I suspected this is what caused that error, but do you know why the order would matter?

Reply:
oh wait I see it now... that `@_objcRuntimeName(_TtC10Foundation13__JSONEncoder)`  is an attribute for `class JSONEncoder`

Reply:
Yes exactly.